src/strands/experimental/bidirectional_streaming/agent/agent.py

```diff
@@ -431,6 +431,75 @@ async def end(self) -> None:
             await stop_bidirectional_connection(self._session)
             self._session = None
 
+    async def __call__(
+        self,
+        send_callable: Callable[[Any], Any],
+        receive_callable: Callable[[], Any],
```

Comment:
Thoughts on:

```suggestion
        *,
        sender: Callable[[Any], Any],
        receiver: Callable[[], Any],
```

That would make this into:

```python
agent = BidirectionalAgent(model=model, tools=[calculator])
agent(sender=websocket.send_json, receiver=websocket.receive_json)
```

My reasoning is:
- Because it requires two parameters, it's non-obvious from the parameter order alone which one is sender which one is receiver; thus the keyword arguments
- simplify name away from `_callable` because I think sender/receiver is more concise/elegant than the repetition of `_callable`